debian/changelog

```diff
@@ -0,0 +1,6 @@
+mgmt (0.0.1-1) unstable; urgency=low
```

Comment:
Awesome dsx! One small comment-- Can we instead generate this from git log, similarly to what is done here: https://github.com/purpleidea/mgmt/blob/master/Makefile#L191 rather than hardcoding this... We don't expect to be the distro upstream package at least in this sort of "commit the changelog" way.